eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -432,17 +432,17 @@ object Helpers {
       Transactions.weight2fee(feeratePerKw, closingWeight)
     }
 
-    def firstClosingFee(nodeParams: NodeParams, commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector)(implicit log: LoggingAdapter): Satoshi = {
+    def firstClosingFee(commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector, feeTargets: FeeTargets)(implicit log: LoggingAdapter): Satoshi = {
```

Comment:
This make the two `firstClosingFee` methods very similar. Would it be a food idea to merge them?

Reply:
I split them to facilitate testing, in [this test](https://github.com/ACINQ/eclair/pull/1066/files#diff-0d859eeb997ad6e4086f3c413770423fR157) i needed to compute the expected fee for the first closing signed negotiation.